www/js/controllers.js

```diff
@@ -4,6 +4,7 @@ angular.module('emission.controllers', ['emission.splash.updatecheck',
                                         'emission.splash.startprefs',
                                         'emission.splash.referral',
                                         'emission.splash.pushnotify',
+                                        'emission.incident.potentialincident.prompt',
```

Comment:
this is not sufficient, you need to inject the service as well.
See `PostTripAutoPrompt` below for an example - otherwise the init code will never be invoked.